ui/admin/app/components/model-providers/ModelProviderForm.tsx

```diff
@@ -119,52 +145,79 @@ export function ModelProviderForm({
             );
 
             await configureModelProvider.execute(
-                modelProviderId,
+                modelProvider.id,
                 allConfigParams
             );
             onSuccess(allConfigParams);
         }
     );
 
     const FORM_ID = "model-provider-form";
+    const showCustomConfiguration =
+        modelProvider.id === "azure-openai-model-provider";
     return (
-        <div className="flex flex-col gap-4">
-            <TypographyH4 className="font-semibold text-md">
-                Required Configuration
-            </TypographyH4>
-            <Form {...form}>
-                <form
-                    id={FORM_ID}
-                    onSubmit={form.handleSubmit(onSubmit)}
-                    className="flex flex-col gap-8"
-                >
-                    {requiredConfigParamFields.fields.map((field, i) => (
-                        <ControlledInput
-                            key={field.id}
-                            label={field.name}
-                            control={form.control}
-                            name={`requiredConfigParams.${i}.value`}
-                            classNames={{
-                                wrapper: "flex-auto bg-background",
-                            }}
+        <div className="flex flex-col">
+            <div className="flex flex-col gap-4 p-4 max-h-[50vh] overflow-y-auto">
```

Comment:
ShadCn comes with a `ScrollArea` component that has a very nicely styled scrollbar. Can you please replace all instances of `overflow[-x|-y]-auto` with it?